### DISCUSSION TITLE
What's the difference between VSCodium, VSCodiumSetup and VSCodiumUserSetup installation files?

### DISCUSSION THREAD

**Discussion by davidjuniorsaylor** [VSCodium/vscodium#1214] [Q&A] [answered]
What's the difference between VSCodium, VSCodiumSetup and VSCodiumUserSetup installation files for Windows? For example:

VSCodium-x64-1.70.2.22230.msi
VSCodiumSetup-x64-1.70.2.22230.exe
VSCodiumUserSetup-x64-1.70.2.22230.exe

**Comment by PolicyPuma** [comment, marked as answer]
I would assume the user setup will install for the local user only, without the need for administrator privileges, whereas the none user setup will install for all users on the system, however it will require administrator privileges.

The `.msi` uses the Microsoft Windows Installer format which is included with Windows itself, you can install VSCodium as a local user or for all users, the latter of which requires administrator privileges. The Microsoft Windows Installer format theoretically is better for compatibility with past and future versions of Windows however it can limit what the developer can do during the installation process. For VSCodium it won't make a difference which installer you choose to use because as far as I'm aware the installation of VSCodium doesn't require anything terribly unique.

For simplicities sake, you can think of `.msi` files as installation files that provide Windows the information to install and manage a piece of software, whereas executables can be just that, executable files that could do anything, they're not limited to software installation however some do provide a delivery of self managed software installation.

**Comment by GitMensch** [comment]
Just to add the official doc: https://code.visualstudio.com/docs/setup/windows#_user-setup-versus-system-setup and (without the explicit link) the note that MS suggests to use the user setup (because of the "smoothness")
